activesupport/lib/active_support/configurable.rb

```diff
@@ -118,7 +120,12 @@ def config_accessor(*names, instance_reader: true, instance_writer: true, instan
             class_eval reader, __FILE__, reader_line if instance_reader
             class_eval writer, __FILE__, writer_line if instance_writer
           end
-          send("#{name}=", yield) if block_given?
+
+          if block_given?
+            send("#{name}=", yield)
+          else
+            send("#{name}=", default)
+          end
```

Comment:
I don't have a strong preference, but what do you think of the more concise:

```suggestion
          send("#{name}=", block_given? ? yield : default)
```

Reply:
I like it! I will apply this change